src/FormMigration/Controllers/MigrationController.php

```diff
@@ -50,6 +51,17 @@ public function __invoke(DonationForm $formV2)
             ->process($payload)
             ->finally(function(FormMigrationPayload $payload) {
                 $payload->formV3->save();
+
+                $campaignRepository = give(CampaignRepository::class);
+
+                // Associate upgraded form to a campaign
+                if ($campaign = $campaignRepository->getByFormId($payload->formV2->id)) {
+                    $defaultForm = $campaign->defaultForm();
+                    $isDefaultForm = $defaultForm && $defaultForm->id == $payload->formV2->id;
```

Comment:
That said, I'm unsure if this part of the code should be responsible for address the case of "no default form".

It might be unexpected behavior for a form migration/upgrade to, in effect, alter the campaign.